.gitmodules

```diff
@@ -1,4 +1,10 @@
 [submodule "tests/googletest"]
-	path = tests/googletest
+	path = third_party/googletest
 	url = https://github.com/google/googletest.git
 	ignore = dirty
+[submodule "third_party/libcudacxx"]
+	path = third_party/libcudacxx
```

Comment:
It would be better to specify the stable releases or branches. These should be the latest version of them, maybe be helpful:
https://github.com/NVIDIA/thrust/releases/tag/2.1.0
https://github.com/NVIDIA/libcudacxx/releases/tag/2.1.0